client/src/TextInput.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+function TextInput() {
+  return (
+    <div>
+      <div className="mt-5 mb-2">
+        <input
+          className="bg-gray-600 placeholder-gray-400 px-5 py-3 rounded text-lg border border-gray-400"
+          type="text"
+          placeholder="Email address"
+        />
+      </div>
+      <div>
+        <input
+          className="bg-gray-600 placeholder-gray-400 px-5 py-3 rounded text-lg border border-gray-400"
```

Comment:
If we take a look at the two divs you're returning here, they are almost identical except for the placeholder property. In the spirit of keeping our code DRY (don't repeat yourself), how can we refactor this code to return a single input but still be reusable for different placeholders?